Monocle/Telegraph.gen.cs

```diff
@@ -6,7 +6,7 @@
 using System.Runtime.Serialization;
 using System.Runtime.InteropServices;
 using System.Reflection;
-
+using System.Text.RegularExpressions;
```

Comment:
I think, in any case because I have always done it and in a project where you directly use the memory of another process as an administrator, I think that this file is judicious in the sense of announcing a vulnerability like its in broad daylight could do some damage, after that is only my opinion.

Reply:
I'm no more expert than her on the subject, it's just that I've always put this file in my repositories by acquired conscience, I also tell myself that a well-configured repository (license, security.md, templates , readme, wiki) may be better valued and referenced by github? However, I cannot confirm this.

Reply:
Done